### PR TITLE
feature/grey-dataset-badges

### PR DESCRIPTION
**Pull request by geeeeemoney**
feature/grey-dataset-badges

### Description

This PR shows all available badges on dataset cards as grey and only highlights them according to the datasets tags

### Checklist

- [x] This PR contains tests (at least one (non-snapshot) test)

### CODE REVIEW

**Review by jhubley — APPROVED** [review]
looks good!

File: src/components/DatasetPreviewCard.tsx [hunk 1]
```diff
@@ -34,7 +35,7 @@ interface IState {}
 
 class DatasetPreviewCard extends React.Component<IProps, IState> {
   render(): ReactElement {
-    const { classes, name, description, updatedAt = 'error', tags = ['is_pii'] } = this.props
+    const { classes, name, description, updatedAt, tags = ['is_pii'] } = this.props
```
Comment (mell0kat) [inline src/components/DatasetPreviewCard.tsx]: Should default tags by `[]`?

File: src/__tests__/components/DatasetPreviewCard.test.tsx [hunk 2]
```diff
@@ -36,11 +36,9 @@ describe('DatasetPreviewCard Component', () => {
         .text()
     ).toContain(formatUpdatedAt(dataset.updatedAt))
   })
-  it('should render a badge per tag that has a corresponding entry in the tagToBadge config', () => {
-    const tagIsInFakeConfig = tag => !!fakeTagToBadge[tag]
-
-    expect(wrapper.find('#tagContainer').children()).toHaveLength(
-      tags.filter(tagIsInFakeConfig).length
+  it('should render a colored and disabled badge', () => {
```
Comment (mell0kat) [inline src/__tests__/components/DatasetPreviewCard.test.tsx]: Not sure this text accurately matches what we're testing?

File: src/__tests__/components/DatasetPreviewCard.test.tsx [hunk 3]
```diff
@@ -1,24 +1,24 @@
-import { mount } from 'enzyme'
+import { shallow, mount } from 'enzyme'
 import * as React from 'react'
 import Typography from '@material-ui/core/Typography'
-import fakeTagToBadge from '../../config/__mocks__/tag-to-badge'
+// import tagToBadge from '../../config/tag-to-badge'
```
Comment (mell0kat) [inline src/__tests__/components/DatasetPreviewCard.test.tsx]: I'm think we should maybe keep the mock config tagToBadge...let's discuss in person

**Review by mell0kat — CHANGES_REQUESTED** [review]

File: src/components/DatasetPreviewCard.tsx (hunk 1 above)
Comment (grantfosterdev) [reply src/components/DatasetPreviewCard.tsx]: yeah I'm down with that, this is legacy from playing around :-|

**Review by mell0kat — APPROVED** [review]

**Pull request merged**